apps/system/test/unit/battery_overlay_test.js

```diff
@@ -129,7 +130,7 @@ suite('battery manager >', function() {
     suite('init >', function() {
       setup(function() {
         MockNavigatorBattery.level = 0.02;
-        subject.start();
+        subject._start();
```

Comment:
Do we know which part of the BaseModule's start method is causing an issue? Can we fix it? Or do we need to call stop() at teardown?